src/main/resources/assets/jade/lang/zh_tw.json

```diff
@@ -77,12 +92,12 @@
 	"config.jade.hide_from_tab_list": "從玩家列表隱藏",
 	"config.jade.hide_from_tab_list_desc": "開啟玩家列表時隱藏物品提示。",
 	"config.jade.hide_from_tab_list_extra_msg": "tab",
-	"config.jade.hide_from_guis": "從GUI隱藏",
+	"config.jade.hide_from_guis": "從 GUI 隱藏",
```

Comment:
Please do not add spaces between English and Chinese, this is not consistent with the formatting in other locations